### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by NarmeenNaveedAhmed**
Release version 5.0.0-rc.3.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,17 @@
-# Changelog
+-e # Changelog
```
Comment (mtomcal) [inline CHANGELOG.md:1]: `-e`?

Comment (nellyk) [reply CHANGELOG.md:1]: might be an issue with the script generator - i''ll review this

Comment (nellyk) [reply CHANGELOG.md:1]: @NarmeenNaveedAhmed  would be removing this manually

Comment (NarmeenNaveedAhmed) [reply CHANGELOG.md]: that was weird! fixed! :)

**Comment by JAdshead** [comment]
make sure to correctly squash commits

**Pull request merged**